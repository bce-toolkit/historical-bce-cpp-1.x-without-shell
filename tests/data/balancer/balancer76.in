Na4XeO6+Mn(OH)2+H2O=NaHXeO4+NaMnO4+NaOH