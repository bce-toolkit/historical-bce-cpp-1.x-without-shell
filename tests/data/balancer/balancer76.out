5Na4XeO6+2Mn(OH)2+7H2O=5NaHXeO4+2NaMnO4+13NaOH